docker-compose.dev.yml

```diff
@@ -1,10 +1,28 @@
 version: "3.7"
 services:
+
+  vespatest:
+    image: vespaengine/vespa:8.273.20
+    container_name: vespatest
+    ports:
+      - 8080:8080
+      - 19071:19071
+    healthcheck:
+      test: curl -f http://localhost:19071/status.html || exit 1
+      interval: 5s
+      timeout: 3s
+      retries: 30
+      start_period: 20s
+
   backend:
     environment:
       PYTHONDONTWRITEBYTECODE: 0
+      VESPA_URL: http://vespatest:19071
```

Comment:
This environment variable needs updating in the .env.example please. Could probably also use an env variable in that file for VESPA_PORT or something as well whilst we're there.

Reply:
Thanks Katy, updating the .env.example now

Reply:
Thanks for the comment on the port as well, after mulling it over, I'd be keen to leave the port as is as for now. It's only needed in dev so I _think_ we'd end up having to add extra complexity in prod/staging to avoid having it in those environments. Also, looking at the other services in .env.example, it seems appending the port directly to the url is consistent with opensearch, etc too, so i'd also rather go for consistency for now and potentially revisit all of them together separate of this change if we decide we want to

Reply:
Happy with this!